app/server/lib/AuditLogger.ts

```diff
@@ -172,15 +207,27 @@ export class AuditLogger implements IAuditLogger {
 
   private async _getOrSetStreamingDestinations(event: AuditEvent) {
     const orgId = event.context.site?.id;
-    const destinations = await Promise.all([
```

Comment:
To recreate similar issue with promises just put those lines here and run the DocApi2 test.
```
      new Promise<AuditLogStreamingDestination[] | null>((resolve) => setTimeout(resolve, 10 * 1000, null)),
      new Promise<AuditLogStreamingDestination[] | null>((res, reject) => setTimeout(reject, 1, null)),
      ```